target/linux/ar71xx/config-4.4

```diff
@@ -113,6 +113,7 @@ CONFIG_ATH79_MACH_GL_INET=y
 CONFIG_ATH79_MACH_GL_MIFI=y
 CONFIG_ATH79_MACH_GS_MINIBOX_V1=y
 CONFIG_ATH79_MACH_GS_OOLITE=y
+CONFIG_ATH79_MACH_HIVEAP_121=y
```

Comment:
As this board belongs to nand subtarget, please enable it only in nand subtarget config-default and disable for other (generic, mikrotik).